Detections/MultipleDataSources/SuspiciousAWSCLICommandExecution.yaml

```diff
@@ -0,0 +1,57 @@
+id: 8c2dc344-9352-4ca1-8863-b1b7a5e09e59

```

Comment:
Given this just uses AWS data it should go in the AWS solution folder rather than in the multiple data sources detection one